shared/chat/inbox/row/small-team-rows.js

```diff
@@ -19,25 +19,28 @@ type SimpleProps = {
   showBold: boolean,
   snippet: string,
   subColor: string,
+  teamname: ?string,
   timestamp: string,
   usernameColor: string,
   youNeedToRekey: boolean,
 }
 
-class SimpleRow extends PureComponent<SimpleProps> {
+class SmallTeamRow extends PureComponent<SimpleProps> {
   render() {
     const props = this.props
     return (
       <ClickableBox onClick={props.onSelectConversation} style={{backgroundColor: props.backgroundColor}}>
         <Box style={{...rowContainerStyle, backgroundColor: props.backgroundColor}}>
-          <Avatars
-            backgroundColor={props.backgroundColor}
-            isMuted={props.isMuted}
-            isSelected={props.isSelected}
-            participantNeedToRekey={props.participantNeedToRekey}
-            participants={props.participants}
-            youNeedToRekey={props.youNeedToRekey}
-          />
+          {props.teamname
```

Comment:
the simple team and adhoc teams basically look the same but very slightly different, just keeping this in one place